content/docs/how-to-guides/using-branches.md

```diff
@@ -0,0 +1,52 @@
+---
+title: How to Use Branches
+---
+
+Now that you have access to [branching](https://neon.tech/docs/conceptual-guides/branching.md) on Neon, let's look at how you can use it.
+
+Branching allows you to create a copy of your data so that you can modify a version of your project data without changing your production data.
+
+A new branch has all the same data that your project had when you first created the branch, but future changes to the data in your project won't update your branch.
+
+Below is a guide to using branching in your project.
+
+Branches currently appear as a whole new project in the Neon UI and API, but when branches become available to all users, you'll be able to find them within their parent project.
+
+---
+Let's look at how you can create branches using either the Neon Dashboard or the Neon API.
+
+Create a Project on Neon and Add Your Data
+===============
+Log in to Neon and create a project on Neon in the UI: https://console.neon.tech
+Add some data using the SQL Editor in the UI, psql, or any other Postgres client.
+
+Our docs have more information about how to create a project :
+https://neon.tech/docs/cloud/getting-started/
+https://neon.tech/docs/cloud/tutorials/
+https://neon.tech/docs/quickstart/postgres/
+
```

Comment:
Maybe just reference this topic:
Setting up a Project
https://neon.tech/docs/get-started-with-neon/setting-up-a-project/